src/main/java/net/neoforged/neoforge/fluids/FluidUnits.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.fluids;
+
+import net.minecraft.resources.ResourceLocation;
+
+public class FluidUnits {
+    public static final FluidUnit BUCKET = c("bucket");
+    public static final FluidUnit BOTTLE = c("bottle");
+    public static final FluidUnit BOWL = c("bowl");
+
+    public static final FluidUnit BLOCK = c("block");
+    public static final FluidUnit INGOT = c("ingot");
+    public static final FluidUnit NUGGET = c("nugget");
+    public static final FluidUnit GEM = c("gem");
+
+    public static final FluidUnit SLIMEBALL = c("slimeball");
```

Comment:
Would use "slime_ball" instead of "slimeball" here, to match the item id of the vanilla slime ball.